github-dark.css

```
@@ -1674,6 +1674,10 @@
     background: #222 !important;
     color: #777 !important;
   }
+  /*Repo name Suggesiton color*/
+  .reponame-suggestion {
+    color: #539c2a !important;
```

```suggestion
    color: #539c2a !important;
```

We have plenty of greens you can add this to, for instance `#55a532` so please add this selector to
https://github.com/StylishThemes/GitHub-Dark/blob/6c61d52cc041c5b4693164754c6ab6ac716d3ff2/github-dark.css#L3498-L3500

You dont need the comment either.